### PR TITLE
Pass nocolor to xunit if archived

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/arcade/issues/2098

cc @wfurt

### CODE REVIEW

**Comment by ViktorHofer** [comment]
Correct.

File: src/Microsoft.DotNet.CoreFxTesting/build/core/Core.targets [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <TestAssembly Condition="'$(TestAssembly)' == ''">$(TargetFileName)</TestAssembly>
     <TestPath Condition="'$(TestPath)' == ''">$(OutDir)</TestPath>
     <RunWorkingDirectory>$(TestPath)</RunWorkingDirectory>
+    <ArchiveTest Condition="'$(ArchiveTests.ToLower())' == 'all' OR $(MSBuildProjectName.EndsWith('.$(ArchiveTests)'))"></ArchiveTest>
```
Comment (safern) [inline src/Microsoft.DotNet.CoreFxTesting/build/core/Core.targets]: Maybe I'm confused here, why are you setting this to empty, and then using ArchiveTest as a boolean?

Comment (ViktorHofer) [reply src/Microsoft.DotNet.CoreFxTesting/build/core/Core.targets]: lol => should be true.

Comment (ViktorHofer) [reply src/Microsoft.DotNet.CoreFxTesting/build/core/Core.targets]: Thanks!!!!!!!!!!!!!!!! 🗡

Comment (safern) [reply src/Microsoft.DotNet.CoreFxTesting/build/core/Core.targets]: 🤣

**Pull request merged**